bin/image-create-azure.sh

```diff
@@ -3,19 +3,18 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
-IMG_VERSION="v040b1"
+[ -z "${IMG_VERSION}" ] && IMG_VERSION="v040b1"
```

Comment:
Please indent the if. If single liner is very bad practice, I am surprised that shellcheck does not complain about it